src/rdf4cpp/storage/identifier/NodeBackendID.hpp

```diff
@@ -6,7 +6,7 @@
 
 namespace rdf4cpp::storage::identifier {
 
-struct NodeBackendID {
+struct alignas(uint64_t) NodeBackendID {
```

Comment:
Why is the alignas necessary? I would have expected that it is already aligned like that.

Reply:
I guess its because of the bitfields, the alignment was 1 before